R/pkg/R/functions.R

```diff
@@ -3062,6 +3077,21 @@ setMethod("array_sort",
             column(jc)
           })
 
+#' @details
+#' \code{arrays_overlap}: Returns true if the input arrays have at least one non-null element in
+#' common. If not and both arrays are non-empty and any of them contains a null, it returns null.
+#' It returns false otherwise.
+#'
+#' @rdname column_collection_functions
+#' @aliases arrays_overlap arrays_overlap,Column-method
+#' @note arrays_overlap since 2.4.0
+setMethod("arrays_overlap",
+          signature(y = "Column", x = "Column"),
```

Comment:
is there a reason this is "y" and then "x" and not "x" and then "y"?

Reply:
Just noticed that "y" is specified as first for other binary methods (e.g. `datediff`, `nanvl`, `shiftLeft`, ...). No serious reason, I can change it.